docs/examples/201/key-vault-secret-create/main.bicep

```diff
@@ -1,3 +1,16 @@
+// 
+// v0.3 Bicep Known issue & work-around related to "secretsObject"
```

Comment:
Let's get rid of this comment assuming we uncomment the default secret object in the array parameter